### DISCUSSION TITLE
HIDS Question

### DISCUSSION THREAD

**Discussion by jdecker79** [Security-Onion-Solutions/securityonion#4470] [Unsupported Versions]
I have two HIDS alerts that account for the majority of my alerts.  60602 (Windows Application error event) is constantly showing because our Google Credential Provider for Windows keeps throwing 256 event logs that have a system.severityValue = ERROR.  I don't want to tune out the rule because if I have a valid event log with system.severityValue = Error I would like to see it.  I just don't want all the ones from Provider GCPW to alert.

The multiple alerts from rule 60602 then triggers alerts from rule 61061 (Multiple Windows error Application events). 

Does anyone have any suggestions on how to filter out the events that are specifically created by Provider: GCPW?

**Comment by weslambert** [comment]
You can filter these out by modifying/suppressing rules in `local.rules`:

https://docs.securityonion.net/en/latest/wazuh.html?highlight=wazuh#tuning-rules
